ts/features/idpay/initiative/details/components/InitiativeCardComponent.tsx

```diff
@@ -23,133 +23,160 @@ type Props = {
-    position: "absolute"
-  },
-  amount: {
-    lineHeight: 32,
-    fontSize: 24,
-    color: IOColors.white,
-    ...makeFontStyleObject("Bold", undefined, "TitilliumWeb")
+    flexDirection: "row",
+    justifyContent: "center"
   },
-  logo: {
-    resizeMode: "contain",
-    height: 65,
-    width: 65
+  bonusStatusContainer: {
+    flexDirection: "row",
+    alignItems: "center"
   },
   consumedOpacity: {
     opacity: 0.5
+  },
+  remainingPercentageSliderContainer: {
+    height: 4,
+    backgroundColor: IOColors.white,
+    width: 100,
+    borderRadius: 4
+  },
+  alignCenter: {
+    alignItems: "center"
   }
 });
 
+type PercentageSliderProps = {
+  percentage: number;
+  isGreyedOut: boolean;
+};
+const BonusPercentageSlider = ({
+  percentage,
+  isGreyedOut
+}: PercentageSliderProps) => {
+  const width = useSharedValue(100);
+  React.useEffect(() => {
+    // eslint-disable-next-line functional/immutable-data
+    width.value = percentage;
+  });
+  const scalingWidth = useAnimatedStyle(() => ({
+    width: withTiming(width.value, { duration: 1000 })
+  }));
+  return (
+    <View style={styles.remainingPercentageSliderContainer}>
+      <Animated.View
+        style={[
+          {
+            width: `${width.value}%`,
+            backgroundColor: isGreyedOut ? IOColors.blue : IOColors["grey-450"],
+            flex: 1,
+            borderRadius: 4
+          },
+          scalingWidth
+        ]}
+      />
+    </View>
+  );
+};
+const formatNumberRightSign = (amount: number) =>
+  `${formatNumberAmount(amount, false)} €`;
 const InitiativeCardComponent = (props: Props) => {
   const { initiativeName, endDate, status, amount, accrued, refunded } =
     props.initiative;
 
   const isInitiativeConfigured = status === InitiativeStatusEnum.REFUNDABLE;
   const toBeRepaidAmount = (accrued || 0) - (refunded || 0);
+  const remainingAmount = (amount || 0) - (accrued || 0);
 
-  const renderFullCard = () => (
-    <View style={[styles.row, styles.spaced, IOStyles.flex]}>
-      <View style={[styles.fullHeight, styles.spaced]}>
-        <View>
-          <VSpacer size={8} />
-          <H2 color="white">{initiativeName}</H2>
-          <Text style={{ color: IOColors.white }}>
-            {I18n.t("idpay.initiative.details.initiativeCard.validUntil", {
-              expiryDate: formatDateAsLocal(endDate, true)
-            })}
-          </Text>
-        </View>
-        <View style={styles.row}>
-          <View>
-            <Text
-              bold={true}
-              style={[
-                !isInitiativeConfigured ? styles.consumedOpacity : {},
-                styles.amount
-              ]}
-            >
-              {formatNumberAmount(amount || 0, true)}
-            </Text>
-            <H5 color="white">
+  const dateString = formatDateAsLocal(endDate, true);
+  const remainingBonusAmountPercentage =
+    amount !== undefined && amount !== 0
+      ? (remainingAmount / amount) * 100.0
+      : 100.0;
```

Comment:
```suggestion
  const remainingBonusAmountPercentage =
    totalAmount !== 0
      ? (remainingAmount / totalAmount) * 100.0
      : 100.0;
```

See comment

Reply:
changes adressed in https://github.com/pagopa/io-app/pull/4436/commits/388f924eb3fa5ea2dc4ec0ab6ace56c49468831c